File: API/QrRollCallAPI/QrRollCallAPI/Models/QR.cs
using System.ComponentModel.DataAnnotations;
namespace QrRollCallAPI.Models
{
    public class QR
    {
        [Key]
        public int qr_no { get; set; }

        [Required]
        public int ak_konum_x { get; set; }

        [Required]
        public int ak_konum_y { get; set; }

        [Required]
        public int hafta { get; set; }

        [Required]
        public int ders_no { get; set; }

        [Required]
        public int ak_no { get; set; }

        [Required]
        public int bolum_no { get; set; }

        [Required]
        public int tarih { get; set; }

        [Required]
        public int saat { get; set; }


    }
}

File: RollcallWithQR/app/src/main/java/com/qrrollcall/qrrollcall/model/qr.java
package com.qrrollcall.qrrollcall.model;

import com.google.gson.annotations.SerializedName;

public class qr {
    @SerializedName("qr_no")
    public int qr_no;

    @SerializedName("ak_konum_x")
    public int ak_konum_x;

    @SerializedName("ak_konum_y")
    public int ak_konum_y;

    @SerializedName("hafta")
    public int hafta;

    @SerializedName("ders_no")
    public int ders_no;

    @SerializedName("ak_no")
    public int ak_no;

    @SerializedName("bolum_no")
    public int bolum_no;

    @SerializedName("tarih")
    public int tarih;

    @SerializedName("saat")
    public int saat;

}
